Library/Homebrew/shims/linux/super/cc

```diff
@@ -150,7 +150,7 @@ class Cmd
       /^-O[0-9zs]?$/, "-fast", "-no-cpp-precomp",
       "-pedantic", "-pedantic-errors", "-Wno-long-double",
       "-Wno-unused-but-set-variable"
-    when "-lintl"
+    when "-lintl", "-liconv"
```

Comment:
Since the order is arbitrary, I'd suggest putting them in alphabetical order. Other than that, looks good to me!